datafusion/core/tests/sqllogictests/test_files/groupby.slt

```diff
@@ -1921,3 +1921,96 @@ SELECT DISTINCT + col1 FROM tab2 AS cor0 GROUP BY cor0.col1
 41
 59
 61
+
+
+
+# Columns in the table are a,b,c,d. Source is CsvExec which is ordered by
```

Comment:
These tests used the same fixture, so I had to port them too if I wanted to remove the get_testdata fixture